src/Application/Users/Models/UserPublicViewModel.cs

```diff
@@ -12,4 +15,10 @@ public record UserPublicViewModel : IMapFrom<User>
     public string Name { get; init; } = string.Empty;
     public Uri? Avatar { get; init; }
     public Region? Region { get; init; }
+    public ClanPublicViewModel? Clan { get; init; } = new();
+    public void Mapping(Profile profile)
```

Comment:
Add a new line between properties and methods

Reply:
done